compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeContainer.uikit.kt

```diff
@@ -380,18 +392,3 @@ private fun getLayoutDirection() =
         UIUserInterfaceLayoutDirection.UIUserInterfaceLayoutDirectionRightToLeft -> LayoutDirection.Rtl
         else -> LayoutDirection.Ltr
     }
-
-@OptIn(InternalComposeApi::class)
-@Composable
-internal fun ProvideContainerCompositionLocals(
```

Comment:
I think it was better - as I said before, it doesn't seem right at all to have `@Composable` functions inside `UIViewController`
Also, you're exposing `interfaceOrientationState` and `systemThemeState` via locals anyway